components/layout_2020/positioned.rs

```diff
@@ -928,33 +943,67 @@ impl<'a> AbsoluteAxisSolver<'a> {
             _ => return None,
         };
 
+        assert_eq!(
+            container_writing_mode.is_horizontal(),
+            abspos_writing_mode.is_horizontal(),
+            "Mixed horizontal and vertical writing modes are not supported yet"
+        );
+
         let mut value_after_safety = self.alignment.value();
         if self.alignment.flags() == AlignFlags::SAFE &&
             margin_box_axis.length > alignment_container.length
         {
             value_after_safety = AlignFlags::START;
         }
 
-        match value_after_safety {
-            AlignFlags::CENTER | AlignFlags::SPACE_AROUND | AlignFlags::SPACE_EVENLY => Some(
```

Comment:
Sorry, I was wrong, content distribution alignments like `space-around` and `space-evenly` are needed when we are using the static position.
Removing them breaks `/css/css-flexbox/abspos/position-absolute-001.html`.

Reply:
No worries, I've added them back